src/PayID/pay-id-client.ts

```diff
@@ -29,28 +42,54 @@ export default class PayIDClient {
     // Swagger generates the '/' in the URL by default and the payment pointer's 'path' is prefixed by a '/'. Strip off the leading '/'.
     const path = paymentPointer.path.substring(1)
 
-    // TODO(keefertaylor): Append headers here.
     const client = new ApiClient()
     client.basePath = `https://${paymentPointer.host}`
-    const apiInstance = new DefaultApi(client)
+
+    // Accept only the given network in response.
+    const accepts = [`application/xrpl-${network}+json`]
 
     return new Promise((resolve, reject) => {
-      // Ignore unused var in generated code.
-      apiInstance.resolvePayID(path, (error, data, _response) => {
-        if (error) {
-          if (error.status === 404) {
-            // Not Found
-            resolve(undefined)
+      // NOTE: Swagger produces a higher level client that does not require this level of configuration,
+      // however access to Accept headers is not available unless we access the underlying class.
+      const postBody = null
+      const pathParams = {
+        path,
+      }
+      const queryParams = {}
+      const headerParams = {}
+      const formParams = {}
+      const authNames = []
+      const contentTypes = []
+      const returnType = Mapping
```

Comment:
Swagger is bad for this problem space haha.

Worth moving this ridiculousness to its own file?

Reply:
At the moment I was going to just leave all the nastiness encapsulated here. When we implement more complex interactions, I was going to see what refactors we could make.